src/main/java/org/openstreetmap/atlas/geography/Polygon.java

```diff
@@ -289,6 +289,32 @@ public int nextVertexIndex(final int currentVertexIndex)
         }
     }
 
+    public boolean overlaps(final MultiPolygon multiPolygon)
```

Comment:
Might be worthwhile adding a test for this method in `PolygonTest`.

Reply:
It is tested in the MultiPolygonTest but that is not obvious. I will move it to that test instead. Thanks!